### PR TITLE
Added a github  enterprise license process map

### PR DESCRIPTION
**Pull request by SHIHO-I**

### CODE REVIEW

File: docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md [hunk 1]
```diff
@@ -32,52 +34,52 @@ For more information on our GitHub organizations and their uses, see [B.C. gover
...
+Once you've been granted access, you'll be able to log in to the B.C. government GitHub Enterprise at [https://github.com/enterprises/bcgov-ent/](https://github.com/enterprises/bcgov-ent/) using your IDIR and credentials begin working in private repositories within your ministry or sector's organization. 
 
-## Pay for the GitHub Enterprise user licences
+## Background: BC government GitHub Enterprise process map
 
-Software Central Management handles your purchase order, with billing back to your expense authority. The ministry is billed monthly for the GitHub Enterprise user licenses until Software Central gets a request asking to cancel the user licenses.
+Our team has created the artifact below which captures the steps and teams involved in acquiring and mananaging GitHub Enterprise licenses. This version is intended for reference only, but we intend to create simplified, more focused versions for both end users and administrators.  
 
+![A detailed diagram illustrating the process of acquiring and administering GitHub Enterprise licenses within the B.C. government. ](../images/github-e-process-map.png)
```
Comment (sheaphillips) [inline docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:79]: Hi @SHIHO-I !

Congratulations on your first pull request!

I fixed the link, moved the location of the diagram and added "alt text", which is important for accessibility.

**Review by sheaphillips — APPROVED** [review]
Hi @SHIHO-I !

I reviewed your addition of the diagram. It looks great. I've made some minor changes to the location in the document and I had to alter the link so it shows up.

Also, I got carried away and ended up rewriting a bunch of the text on the page. Please take a look and let me know your thoughts. I will share with @jenreiher-bcgov  also.

File: docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ sort_order: 4
 
 # GitHub Enterprise user licences in the BC government
 
+>Note: This page is under review as of December 2023 and will be updated for clarity and accuracy early in 2024.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:23]: if this is still here in June, please lodge a formal compliaint with the management

File: docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md [hunk 3]
```diff
@@ -20,6 +20,8 @@ sort_order: 4
 
 # GitHub Enterprise user licences in the BC government
 
+>Note: This page is under review as of December 2023 and will be updated for clarity and accuracy early in 2024. 
+
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:24]: ## When should you put your code in GitHub Enterprise

File: docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md [hunk 4]
```diff
@@ -20,6 +20,8 @@ sort_order: 4
...
 All code by B.C. government teams should be open source by default. If you have closed-source code and still want to use GitHub, you can work temporarily in a private repository under the GitHub Enterprise licence.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:25]: All code by B.C. government teams should be open source by default. If you have closed-source code and still want to use GitHub, you can work temporarily in a private repository within th BC Gov-managed [GitHub Enterprise organization](https://github.com/enterprises/bcgov-ent).

File: docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md [hunk 5]
```diff
@@ -32,52 +34,52 @@ For more information on our GitHub organizations and their uses, see [B.C. gover
...
 
-User licenses for accessing the private repositories come from a central pool under the GitHub Enterprise account and must be purchased for each user that needs access to the private repositories in the ministry’s private organizations.  Management of this pool of seats is done in partnership with each sector or ministry purchasing their own allocated seat count.
+Other details of GitHub Enterprise include the following:
 
-When your private organization is linked to the GitHub Enterprise account certain settings are automatically enabled or disabled. For example, the ability to create public repositories within this private organization is removed to fall in line with all Provincial open-source code remaining in `bcgov`. Other features include the following:
+* Single sign-on is provided via Azure AD (aka Entra ID)
+* Login uses government-issued IDIR identities
+* Users will be automatically assigned new GitHub IDs, distinct from public GitHub IDs
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:47]: Users will be automatically assigned new GitHub IDs, changing login-in flow for teams using both public and private repositories

File: docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md [hunk 6]
```diff
@@ -32,52 +34,52 @@ For more information on our GitHub organizations and their uses, see [B.C. gover
...
+## GitHub Enterprise account ownership
 
-## Request GitHub Enterprise user licences
+The B.C. government's GitHub Enterprise account is managed centrally by the Digital Office's [Developer Experience Team](mailto:developer.experience@gov.bc.ca). Ministries or sectors are provided with GitHub "organizations" within the central GitHub Enterprise account. Teams' private repositories are created within these organizations.  Management of ministry or sector organizations within GitHub Enterprise is delegated to staff within the corresponding IMB/ISB. 
 
-Contact Software Central Management at SoftwareCentral.Management@gov.bc.ca and cc Dean Picton (Dean.Picton@gov.bc.ca). Use the subject line "GitHub Enterprise Request" and send the following information:  
+Paid licenses are required to use GitHub Enterprise. Licenses must be purchased for each user that will use or administer to the B.C. government GitHub Enterprise organizations or repositories. Each ministry or sector is responsible for acquiring and paying for its own licenses. The process for acquiring licences is described below.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:55]: link?

File: docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md [hunk 7]
```diff
@@ -32,52 +34,52 @@ For more information on our GitHub organizations and their uses, see [B.C. gover
...
 
-* GitHub IDs of the organization administrators
+## Acquiring GitHub Enterprise user licences
 
-* Number of user licenses needed. If you're transferring a GitHub organization to Enterprise, request as many user licenses as you have organization members.
+> Note: the process outlined below may be slightly different for some sectors or ministries, but it will be generally similar for all. For example, in some areas, the responsibility for interacting with CSAM may be done by the ministry or sector IMB/ISB staff rather than end users in a business area.
 
-## Access the GitHub Enterprise account
+Licenses for GitHub Enterprise must be purchased through OCIO Software Central Management. If you have determined that you or others in your area require licenses, the first step is to request a quote via email to  [SoftwareCentral.Management@gov.bc.ca](mailto:SoftwareCentral.Management@gov.bc.ca). Use the subject line "Request for Quotation for GitHub Enterprise"  and indicate in the body the number of users who need GitHub Enterprise licenses. For example, "Could I please get a quote for 10 GitHub Enterprises licenses (Standalone)?".
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:63]: 1. Determine your team's needs for licenses. 
2. request a quote via email to  [SoftwareCentral.Management@gov.bc.ca](mailto:SoftwareCentral.Management@gov.bc.ca). Use the subject line "Request for Quotation for GitHub Enterprise"  and indicate in the body the number of users who need GitHub Enterprise licenses. For example, "Could I please get a quote for 10 GitHub Enterprises licenses (Standalone)?".
3. Work with your business area to determine who will have the responsibility to create an iStore order when the licenses are ready to be assigned. 
4. Once you have received a response from CSAM with a quote number and the cost, initiate the purchase & e-approvals process via iStore. 
5. Wait for CSAM to notify you by email once they have made the purchase on your behalf. 
6. Billing back to your expense authority will commence. Billing occurs monthly until you notify CSAM to cancel the user licenses.
7. Your ministry administrator will connect your subscription to and IDIR together via Azure AD to grant you access to GitHub.

File: docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md [hunk 8]
```diff
@@ -32,52 +34,58 @@ For more information on our GitHub organizations and their uses, see [B.C. gover
...
 
-* Two-factor authentication is mandatory
-* GitHub Actions is enabled for all
-* Code Dependency Insights is enabled for all organizations linked to the GitHub Enterprise account
+## GitHub Enterprise account ownership
 
-## Request GitHub Enterprise user licences
+The B.C. government's GitHub Enterprise account is managed centrally by the Digital Office's [Developer Experience Team](mailto:developer.experience@gov.bc.ca). Ministries or sectors are provided with GitHub "organizations" within the central GitHub Enterprise account. Teams' private repositories are created within these organizations.  Management of ministry or sector organizations within GitHub Enterprise is delegated to staff within the corresponding IMB/ISB. 
 
-Contact Software Central Management at SoftwareCentral.Management@gov.bc.ca and cc Dean Picton (Dean.Picton@gov.bc.ca). Use the subject line "GitHub Enterprise Request" and send the following information:
```
Comment (SHIHO-I) [inline docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:53]: @sheaphillips @jenreiher-bcgov The sentence "Use the subject line "GitHub Enterprise Request" and send the following information:" has a slightly different recommended email subject from Steps. Also, as no following information is available here, is that better to say something like "Please refer to the "Steps" section below."? What do you think?

Comment (SHIHO-I) [inline docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:53]: @sheaphillips @jenreiher-bcgov The sentence "Use the subject line "GitHub Enterprise Request" and send the following information:" has a slightly different recommended email subject from Steps. Also, as no following information is available here, is that better to say something like "Please refer to the "Steps" section below."? What do you think?

Comment (jenreiher-bcgov) [reply docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md:53]: great catch - I'll make that update. thank you!

**Pull request merged**